18-interrupts/cpu/idt.h

```diff
@@ -27,10 +27,6 @@ typedef struct {
     u32 base;
 } __attribute__((packed)) idt_register_t;
 
-#define IDT_ENTRIES 256
```

Comment:
This pull request does not contain change of moving deleted code into c file.

Reply:
Ok, I see your changes spreaded into multiple pull requests (https://github.com/cfenollosa/os-tutorial/pull/181/files).